pkgs/by-name/op/opensta/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  tcl,
+  swig,
+  bison,
+  flex,
+  git,
+  cudd,
+  eigen,
+  libzip ? null,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "opensta";
+  version = "0-unstable-2025-04-24";
+
+  src = fetchFromGitHub {
+    owner = "parallaxsw";
+    repo = "OpenSTA";
+    rev = "cba0cf5cc43f5ba484bfe8e80f137a6d71dff56e";
+    sha256 = "1v8xicv5syp3x6gn5f2rdn9rdg9pj6x60qdkdik8izzc25s2cjac";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    bison
+    flex
+    git
+    eigen
+    cudd
+    swig
+  ];
+
+  buildInputs = lib.optional stdenv.isLinux libzip ++ [ tcl ];
```

I fear you may have mixed up a few build and run‐time dependencies. This often works for native builds because the default `stdenv` is very lax, but it will probably break cross builds. Enabling `strictDeps` is very helpful to diagnose the issues.

```suggestion
  strictDeps = true;

  nativeBuildInputs = [
    cmake
    bison
    flex
    git
    swig
    tcl
  ];

  buildInputs = lib.optional stdenv.isLinux libzip
    ++ [
      cudd
      eigen
      flex
      tcl
    ];
```